app/controllers/super_admin/wifi_user_search_controller.rb

```diff
@@ -6,7 +6,7 @@ def search
       @wifi_user = WifiUser.search(search_term.gsub(' ', ''))
     end
 
-    render action: 'lookup'
+    render 'logs_searches/contact'
```

Comment:
you are missing `action`, is that an issue?

Reply:
Not in this case as it's telling Rails to render a particular ERB partial.
https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials